plugins/inventory/vmware_vm_inventory.py

```diff
@@ -69,67 +69,69 @@
             description:
             - Specify the list of VMware schema properties associated with the VM.
             - These properties will be populated in hostvars of the given VM.
-            - Each value in the list specifies the path to a specific property in VM object.
+            - Each value in the list can be a path to a specific property in VM object or a path to a collection of VM objects.
+            - Set value to 'all' to query all properties
+            - For all properties refer following URL
+            - See U(https://github.com/monkey-mas/lab/blob/master/pyvmomi/docs/vim/VirtualMachine.rst#attributes).
```

Comment:
I know that VMware repo removed this page, but I would prefer something stays with the Ansible codebase / documentation.

Reply:
Or using `Properties: [all]` to list all and choose from it. Because properties keep evolving from , vCenter|vSphere, version to version.

Reply:
@Akasurde any suggestion on this ?